editor.planx.uk/src/@planx/components/Send/model.ts

```diff
@@ -25,7 +25,7 @@ export const USER_ROLES = ["applicant", "agent", "proxy"] as const;
 // See minimum POST schema for /api/v1/planning_applications
 // https://ripa.bops.services/api-docs/index.html
 interface BOPSMinimumPayload {
-  application_type: "lawfulness_certificate";
+  application_type: "lawfulness_certificate" | string;
```

Comment:
i can never recall if there is a nicer way to indicate a "default" value in a type interface? this syntax feels clumsy.